### PR TITLE
Integrate the current driver with the bblfsh-sdk

### PR DESCRIPTION
**Pull request by alcortesm**
Rebased for your convenience.

### CODE REVIEW

**Comment by ajnavarro** [comment]
should Travis pass?

**Comment by alcortesm** [comment]
Yes, it should.  I think it is not passing because this has been generated with an outdated version of the SDK, let me fix that.

**Comment by alcortesm** [comment]
Rebased.

**Comment by alcortesm** [comment]
huzzah, green!

File: Dockerfile.build.tpl [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Dockerfile.build represents the build environment of the driver, used during
```
Comment (smola) [inline Dockerfile.build.tpl]: Please, remove all comments from the template and leave only any comment you need that is specific to your Dockerfile, or remove comments altogether.

File: Dockerfile.tpl [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Dockerfile represents the container being use to run the driver, should be
```
Comment (smola) [inline Dockerfile.tpl]: Please, remove all comments from the template and leave only any comment you need that is specific to your Dockerfile, or remove comments altogether.

File: manifest.toml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Manifest contains metadata about the driver. To learn about the different
```
Comment (smola) [inline manifest.toml]: Please, remove all comments from the template and leave only any comment you need that is specific to your manifest, or remove comments altogether.

File: Dockerfile.build.tpl (hunk 1 above)
Comment (alcortesm) [inline Dockerfile.build.tpl]: Done.

File: Dockerfile.tpl (hunk 2 above)
Comment (alcortesm) [inline Dockerfile.tpl]: Done.

File: manifest.toml (hunk 3 above)
Comment (alcortesm) [inline manifest.toml]: Done.

**Comment by smola** [comment]
@alcortesm Please clean up history.

**Comment by alcortesm** [comment]
Done.

**Pull request merged**